src/ir/var.rs

```diff
@@ -146,6 +146,11 @@ impl ClangSubItemParser for Var {
                 let (type_kind, val) = match value {
                     EvalResult::Invalid => return Err(ParseError::Continue),
                     EvalResult::Float(f) => {
+                        // Bail out if this number is not finite. We could
+                        // generate those, I guess, but its utility is unclear.
```

Comment:
I guess I'd flip this question on its head: why wouldn't we want to generate bindings for them? That's kinda the business we're in ;-P

File a follow up? Maybe it would be a good first bug?